ibm/mas_devops/roles/cp4d_service/templates/services/ca.yml.j2

```diff
@@ -0,0 +1,15 @@
+apiVersion: ca.cpd.ibm.com/v1
+kind: "{{ cpd_service_info[cpd_service_name].crd_kind }}"
+metadata:
+  name: "{{ cpd_service_info[cpd_service_name].cr_name }}"
+  namespace: "{{ cpd_instance_namespace }}"
+spec:
+  blockStorageClass: "{{ cpd_service_block_storage_class }}"
+  fileStorageClass: "{{ cpd_service_file_storage_class }}"
```

Comment:
same as above comments, can we make it like other cpd roles that use file storage class and have just `cpd_service_storage_class ` as it's normally auto set to the defaulted file storage class for the targeted cloud provider?

Reply:
@andrercm Let's discuss this change in more detail.

Reply:
Removed fileStorageClass based on discussion with @andrercm